src/scripts/main.js

```diff
@@ -354,7 +354,37 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard')
 
-// write your code here
+for (const person of people) {
+  const tableRow = document.createElement('tr');
+  const fragment = document.createDocumentFragment();
+  const tableData = document.createElement('td');
+
+  const name = tableData.cloneNode(true);
+  name.textContent = person.name;
+  fragment.appendChild(name);
+
+  const gender = tableData.cloneNode(true);
+  gender.textContent = (person.sex === 'm') ? 'Male' : 'Female';
+  fragment.appendChild(gender)
```

Comment:
There is a missing semicolon at the end of this line. While JavaScript often handles missing semicolons gracefully, it's a good practice to include them to avoid potential issues in different environments.